History

michielbdejong opened the pull request.

michielbdejong committed:
First version of work delivered by Rajnish, see #823

michielbdejong committed:
We don't have a newsletter

michielbdejong committed:
No more use for DependaBot in this repo

michielbdejong committed:
Rajnish work rounds 2 and 3

michielbdejong committed:
linebreaks in readme

michielbdejong committed:
Update README.md

michielbdejong committed:
Convert all html and css file line endings to LF instead of CRLF, as recommended for git-handled repos

michielbdejong committed:
Keep old tagline

michielbdejong committed:
Small corrections

michielbdejong committed:
improvements by @TallTed

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

michielbdejong committed:
improvements by @TallTed

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

michielbdejong committed:
improvements by @TallTed

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

michielbdejong committed:
improvements by @TallTed

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

michielbdejong committed:
improvements by @TallTed

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

michielbdejong committed:
improvements by @TallTed

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

michielbdejong committed:
improvements by @TallTed

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

michielbdejong committed:
improvements by @TallTed

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

michielbdejong committed:
Improvement by @TallTed

michielbdejong committed:
Explain use of file URLs

For website development

michielbdejong committed:
reword

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

VirginiaBalseiro committed:
Refactor Redesign Code (#836)

* WIP: refactor HTML/CSS

Co-authored-by: Sarven Capadisli <info@csarven.ca>

* HTML corrections / alignment for index, about

* consolidate stylesheets, minor fixes

* Update homepage img alt for article

* Minor

* Remark for-developers

* Remark for-organizations

* Add contact possibilities for orgs

* Update pod-provider and research-institution blocks

* borders, minor CSS changes

* Use details for whole video figcaption

* Minor

* CSS fixes, WIP

* Add Terms

* Minor

* Update community

* Use /about

* Add redirects file

* Add some DOAP about Solid Project

* Remove unused reference

* Add ethicsPolicy to Solid CoC

* WIP styles for community page, mobile

* Minor

* CSS fixes

* Adjustments to events details margin/padding

* Padding, lists styles

* Padding

* Update hosted-pod-services listing markup

* Update events markup

* Hide dt in hosted-pod-services

* fix about page file extension, inline widths, image alt text and role

* Adjust font-size in pod providers list

* Update alt/role

* Minor

---------

Co-authored-by: Sarven Capadisli <info@csarven.ca>

michielbdejong committed:
Fix some broken links, ref #845

michielbdejong committed:
Broken link about_solid.html

michielbdejong committed:
Fix license link

michielbdejong committed:
Fix two buttons that point nowhere

michielbdejong committed:
Point to Discourse and Matrix, fix https://github.com/solid/team/issues/61

michielbdejong commented:
Need to fix the layout

TallTed commented on history.html:
```diff
@@ -0,0 +1,204 @@
+            Pru d'hommeaux, E., 2007. Tabulator redux: Writing into the
+            semantic web</p>
+        
+          <p>2009: Tim BL gave a Ted talk about the importance of the
```
```suggestion
          <p>2009: Tim BL gave a TED Talk about the importance of the
```

TallTed reviewed (commented):
There's a *lot* of broken HTML in here. Just one example — `make a class="c11"&gt;read-write web of data.` — note the missing `<` to open the anchor, and the incorrectly escaped `&gt;` to close that opener, and then the complete lack of anchor closure.

KyraAssaad committed:
Add button-purple class

KyraAssaad committed:
Revert "Add button-purple class"

This reverts commit 0ba79a5adf970acd521803ce7f2f6d71be9a42c0.

megoth committed:
Adding and changing some URLs

Based on feedback from Kyra, which I agree to

michielbdejong committed:
Add Solid World Feb 2024

michielbdejong committed:
Import https://raw.githubusercontent.com/solid/solidproject.org/db1632f795ddc8dca376f9b545675a1f7af8f535/pages/solid-events.md

michielbdejong committed:
markdown to javascript

michielbdejong committed:
Script to generate community.html

michielbdejong committed:
events migration script

michielbdejong committed:
Write community.html with script

michielbdejong committed:
Fix details

michielbdejong committed:
hand-fix 2024 events

michielbdejong committed:
remove hyphens after dates

michielbdejong committed:
Convert remaining markdown links

michielbdejong committed:
Fix hyphen in script

michielbdejong committed:
</time>–</dd>

michielbdejong committed:
A few more manual fixes

michielbdejong committed:
remove bogus event in 2021

michielbdejong committed:
remove migration folder

megoth committed:
Added stubs to ensure old URIs for old website

Some resources are removed altogether though, and I don't know where to point to on the new website.

csarven committed:
Add for-users page

Resolves https://github.com/solid/solidproject.org/issues/859

csarven committed:
Add missing for-users page

csarven committed:
Fix misalignment on homepage and border-bottom none for arbitrary items

Resolves https://github.com/solid/solidproject.org/issues/866

csarven committed:
Fix invalid HTML

csarven committed:
Fix events grid layout

Closes https://github.com/solid/solidproject.org/issues/858 https://github.com/solid/solidproject.org/pull/867

csarven committed:
Minor

michielbdejong committed:
Align home page li text indentation, ref #866

michielbdejong committed:
Update tutorials list, fix #850

michielbdejong committed:
Fix lorem ipsum text for BBC

michielbdejong committed:
Add Athumi, Inrupt, TrinPod, Digita, O-Team, Ponder Source

michielbdejong committed:
Fix logos

michielbdejong committed:
Download and unzip artifact from https://github.com/solid/solidproject.org/actions/runs/8885886631/job/24398233370

michielbdejong committed:
a draft version -> historical content

michielbdejong committed:
Remove files that will be overwritten by the redesign, but keep the rest

michielbdejong committed:
Stab at https://github.com/solid/solidproject.org/issues/894#issuecomment-2144677278

michielbdejong committed:
Add empty Gemfile to satisfy Netlify

michielbdejong committed:
Gemfile from main branch

michielbdejong committed:
Remove bundler version check

ewingson committed:
update reference ... link and link text ... from example.org to current URL ...

d2024-06-03 t04:56

ewingson committed:
update solidweb.me signup link to current v7 URL

d2024-06-03 t06:33

michielbdejong committed:
Merge pull request #895 from ewingson/patch-1

update reference redpencil

michielbdejong committed:
Add history.html

csarven committed:
Updates markup, stylesheet, links/references, RDFa, editorial and content changes

csarven commented:
@TallTed @michielbdejong There were way too many changes to do such that it was easier to push this commit https://github.com/solid/solidproject.org/pull/847/commits/b0247f1e7dbefd4a0d153ffe724c0e7c3ce501e6 (including "markup, stylesheet, links/references, RDFa, editorial and content changes") instead of making inline suggestions through GitHub's UI. It is a take it or leave it.

@TallTed I've done some minor editorial changes as well as proposing changes mentioned in https://github.com/solid/solidproject.org/issues/840#issuecomment-1991391596 as an initial pass I think you should continue to review this article and also perhaps check for accuracy / relevancy. The article still seems like a draft but it can be improved - whether through your and/or others changes (and over time).

@timbl Have a look.

csarven committed:
Add markup for time

michielbdejong commented:
rendering is still not great but better than nothing I think: 
<img width="1528" alt="Screenshot 2024-06-13 at 11 17 23" src="https://github.com/solid/solidproject.org/assets/408412/921405b7-182f-4770-b6c7-f77090710b87">

As a possible improvement, do you think you could center the white rectangle and maybe give it rounded corners, like on the main /about.html page?

Only real rendering bug I see is the aspect ratio of the avatar in the 'More details about this document' view, it's very wide and not very high:
<img width="1528" alt="Screenshot 2024-06-13 at 11 20 20" src="https://github.com/solid/solidproject.org/assets/408412/e6cd2021-b7e8-4419-a800-fad2984b0efb">

michielbdejong commented:
Here is that same avatar but in its correct aspect ratio: https://www.w3.org/Press/Stock/Berners-Lee/2001-europaeum-eighth.jpg

csarven committed:
Minor. Link to TR/protocol

jeswr commented:
@michielbdejong are you still wanting to pursue this PR?
